en/installation-guide/workstation.rst

```diff
@@ -20,13 +20,13 @@ Set the environment variable ``PGSYSCONFDIR`` to the path to ``PGSYSCONFDIR``.
 
 .. note:: Search for environment variable in your windows system settings, then click one of the following options, for either a user or a system variable:
 
-    Click New to add a new variable name and value.
-    Click an existing variable, and then click Edit to change its name or value.
-    Click an existing variable, and then click Delete to remove it.
+    Click ``New`` to add a new variable name and value.
+    Click an existing variable, and then click ``Edit`` to change its name or value.
+    Click an existing variable, and then click ``Delete`` to remove it.
 
   .. figure:: images/umgebungsvariablen_pgconfdir.jpg
   
-  You can check your environment variables also within QGIS: Menu Settings - Options..., System Tab -> Environment
+  You can check your environment variables also within QGIS: ``Menu Settings`` - ``Options...``, ``System Tab`` -> ``Environment``
```

Comment:
That should probably be

```suggestion
  You can check your environment variables also within QGIS: Menu ``Settings`` - ``Options...``, ``System`` Tab  -> ``Environment``
```

Reply:
@kant I can't include those changes myself. It would be great if you could review these and accept if you are ok with it.